symfit/core/fit.py

```diff
@@ -568,19 +638,32 @@ def _init_constraints(self, constraints, model):
                     )
         return con_models
 
+    @keywordonly(callback=False)
     def execute(self, **minimize_options):
         """
         Execute the fit.
 
         :param minimize_options: keyword arguments to be passed to the specified
             minimizer.
+        :param callback: keyword only argument that will be propagated to the
```

Comment:
This kwargs does not make sense to me. A callback has to be a callable. I think what you're going for is the keyword 'verbose'?